emcc.py

```diff
@@ -1311,6 +1312,9 @@ def default_setting(name, new_default):
     for key in settings_map:
        if key not in COMPILE_TIME_SETTINGS:
          diagnostics.warning('unused-command-line-argument', "linker setting ignored during compilation: '%s'" % key)
+    for flag in LINK_TIME_FLAGS:
+        if flag in state.orig_args:
+         diagnostics.warning('unused-command-line-argument', "linker setting ignored during compilation: '%s'" % flag)
```

Comment:
I'm not sure if it's the new github UI, but is indentation 2 spaces here? Seems odd...

Reply:
It's me :)